changelog/2.072.0_pre.dd

```diff
@@ -124,30 +124,32 @@ $(HR)
 $(BUGSTITLE Language Changes,
     $(LI $(LNAME2 deprecated_commaexp, Using the result of a comma expression is deprecated.)
 
-    $(P Comma expressions expressions have proven to be a frequent source of confusion, and bug.
-        Using their result will now trigger a deprecation message.)
+    $(P Comma expressions have proven to be a frequent source of confusion, and bugs -
+    see $(LINK2 https://issues.dlang.org/show_bug.cgi?id=2659#c2, here) for examples.)
```

Comment:
@andralex I think all of myself, @aG0aep6G, @wilzbach and anyone with an opinion agrees that `HTTPS` and such are harmful and should be replaced. An argument I haven't seen mentioned is that with `LINK2` etc. the link appears as it is in the source code in the same way as it will be in HTML and the browser URL. With `HTTP`/`HTTPS` it's no longer an URL, but some macro cacophony that we have created. This makes e.g. grepping for links harder. There is also the semantic issue that it moves the URL protocol, which is part of the URL (a self-contained reference to something) into a macro (which we use semantics for), thus it moves information across a boundary that should be left alone. The 6-7 characters saved in exchange make a poor contrary position, considering the length of typical URLs. As such, I recommend dropping this argument and converting the `HTTP`/`HTTPS` macros to `LINK2` or whatever whenever convenient.